ql/src/java/org/apache/hadoop/hive/ql/plan/CreateDatabaseDesc.java

```diff
@@ -37,6 +38,7 @@
   String comment;
   boolean ifNotExists;
   Map<String, String> dbProperties;
+  ReplicationSpec replicationSpec;
```

Comment:
replicationSpec is unused. Can be removed.

Reply:
ok